src/components/Iframe/index.vue

```diff
@@ -30,6 +30,11 @@
         required: false,
         default: 'camera *; geolocation *; microphone *; autoplay *'
       },
+      allowFullScreen : {
```

Comment:
i think, `allow ` prop already did that, you can add it like `allow="fullscreen"`

Reply:
but if so, you need to add it as boolean type instead of string.
check out it here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#Attributes

Reply:
I did try that but when the youtube video shows the fullscreen option is greyed out. I wasn't until I added the allowFullScreen that I was able to get fullscreen working.